src/main/java/core/basesyntax/DataRangeValidator.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DataRangeValidator {
```

Comment:
Better create private method inside class SalaryInfo for checking dates.
I do not think we need this separate class